### PR TITLE
Adds key usages and update usage certificate yaml

### PR DESCRIPTION
**Pull request by JoshVanL**
Signed-off-by: JoshVanL <vleeuwenjoshua@gmail.com>
/assign @munnerz

### CODE REVIEW

File: content/en/docs/usage/certificate.md [hunk 1]
```diff
@@ -33,14 +34,30 @@ metadata:
   name: example-com
   namespace: default
 spec:
+  # Secret names are always required.
   secretName: example-com-tls
   duration: 2160h # 90d
   renewBefore: 360h # 15d
+  organization: jetstack
```
Comment (munnerz) [inline content/en/docs/usage/certificate.md]: This should be `organizations:` and is a `[]string`

Comment (JoshVanL) [reply content/en/docs/usage/certificate.md]: Changed, but it is actually `organization` in the API :man_shrugging: 


```
            organization:
              description: Organization is the organization to be used on the Certificate
              type: array
              items:
                type: string
```

**Comment by JoshVanL** [comment]
back to you

File: content/en/docs/usage/certificate.md (hunk 1 above)
Comment (munnerz) [reply content/en/docs/usage/certificate.md]: 🤦‍♂

Comment (munnerz) [reply content/en/docs/usage/certificate.md]: This might be a candidate to fix up in a future version...

**Pull request merged**